packages/expo-speech/src/Speech/Speech.ts

```diff
@@ -61,6 +66,13 @@ export function speak(text: string, options: SpeechOptions = {}) {
   ExponentSpeech.speak(String(id), text, options);
 }
 
+export async function getAvailableVoicesAsync(): Promise<any> {
```

Comment:
could you add an interface for `Voice` objects and change the return type to `Promise<Voice[]>`?